Ginger/GingerCore/Resources/GingerHTMLHelper.js

```diff
@@ -118,6 +118,43 @@ function define_GingerLib() {
             var txt = PayLoad.GetValueString();
             return GingerLib.Echo(txt);
         }
+        else if (Command === "WidgetsUIElementAction")
+        {
+           var list = PayLoad.GetListPayLoad();
+           var inputValues = PayLoad.GetInputValues(list);
+
+           var ElementAction = inputValues["ElementAction"];
+           var ElementLocateBy = inputValues["ElementLocateBy"];
+           var ElementLocateValue = inputValues["ElementLocateValue"];
+           var Value = inputValues["Value"];
+
+            if (ElementAction == "TriggerJavaScriptEvent")
+            {
+                var IsMouseEvent = inputValues["IsMouseEvent"];
+
+                var element = GingerLib.GetElement(ElementLocateBy, ElementLocateValue);
+
+                if (IsMouseEvent.toLowerCase() == "true")
```

Comment:
Need to add validation if element is null then return error payload like we are doing inside Handle HTML Elementcontrol action